worker/vcs/manager.go

```diff
@@ -59,9 +59,21 @@ func insert(url, insertion string, index int) string {
 func getUrlwithToken(url, subvcs, token string) string {
 	switch subvcs {
 	case api.GITHUB:
+		if token == "" {
+			log.Warn("subvcs is github, but the token is empty, there meybe exist error")
```

Comment:
I think the log could be "Sub VCS is GitHub, but the token is empty, maybe there is an error."